runtime/cudaq/qis/execution_manager.h

```diff
@@ -117,7 +118,10 @@ class ExecutionManager {
   virtual void initializeState(const std::vector<QuditInfo> &targets,
                                const void *state,
                                simulation_precision precision) = 0;
-
+  /// @brief Initialize the state of the given qudits to the provided
+  /// simulation state.
+  virtual void initializeState(const std::vector<QuditInfo> &targets,
+                               const SimulationState *state) = 0;
```

Comment:
Do you want to provide the overload
```c++
virtual void initializeState(const std::vector<QuditInfo> &targets,
                               const SimulationState &state) {
    initializeState(targets, &state);
}
```

Reply:
Good idea. Added in https://github.com/NVIDIA/cuda-quantum/pull/1582/commits/97c8317595e1c473b4411a9a5338b62eb46bd8f1